readme.md

```diff
@@ -3,14 +3,26 @@ LÖVE is an *awesome* framework you can use to make 2D games in Lua. It's free,
 [![Build Status: Windows](https://ci.appveyor.com/api/projects/status/chc0hdr08wv1d5c7?svg=true)](https://ci.appveyor.com/project/AlexSzpakowski/love)
 [![Build Status: Github CI](https://github.com/love2d/love/workflows/continuous-integration/badge.svg)](https://github.com/love2d/love/actions?query=workflow%3Acontinuous-integration)
 
-Documentation
--------------
+Please see our [living wiki documentationwiki][wiki] for deeper explanations and documentation of this framework.
 
-We use our [wiki][wiki] for documentation.
-If you need further help, feel free to ask on our [forums][forums], our [Discord server][discord], or our IRC channel [#love on OFTC][irc].
+Come join us on our [forums][forums], our [Discord chat][discord], or our IRC channel [#love on OFTC][irc] and feel free to ask us anything.
 
-Compilation
------------
+Contributing
+------------
+
+Please contribute by looking through the [open issues][issues] and the forums and chat.
+Ready to make a code contribution? Very welcome! Please also read the [source code style guide][codestyle].
```

Comment:
"Very welcome!" as a phrase sounds a bit weird to me... either something like "You're welcome to do so!" or "We'd be glad!" or something...

Reply:
Agreed. I think we can find better phrasing, to make it feel like newconers are very welcomed. The original text works as long as PRs are really welcome and users don't get too hindered by the code style requirements. Automatic code style checks is a good way to solve that, and maybe style checks are already automated?

Reply:
I ended up not changing this wording.

Reply:
Either grammatically or semantically, it's still wrong. What's exactly welcome? The user? The changes made? I'm sure the last thing we want is people to find issue with the non-englishness of the readme file. :3

Reply:
Of course, so right you are. I'm not a native English speaker and I'm bound to make mistakes. I also do value that many potential contributors are not native English speakers, so we would be wise to take a constructive stance towards making it easy for people from all walks of life to take part and contribute. The first step towards such a goal is to establish a welcoming atmosphere.
Please, do feel free to give constructive suggestions on how to further improve on the choice of words, grammar, structure, etc. and to your hearts content! Pray, do have a very nice day!

Reply:
The new sentence in question is, since some hours ago, changed to read like this:

> Ready to make a code contribution? Excellent! Please come discuss any bigger changes in the chat, and do also check our [source code style guide][codestyle].

Maybe there's an implicit "and" in there that I ought to make explicit? What's your take?